sparsecoding/inference/iht.py

```diff
@@ -9,25 +9,23 @@ class IHT(InferenceMethod):
     Infer coefficients for each image in data using elements dictionary.
     Method description can be traced to
     "Iterative Hard Thresholding for Compressed Sensing" (T. Blumensath & M. E. Davies, 2009)
+
+    Parameters
+    ----------
+    sparsity : scalar (1,)
```

Comment:
is there a reason to show the size for scalar and string (line 20)? if so, should do consistently across docstrings. if not, can remove (can do this in a separate pr if it's too much for this one)

Reply:
Fixed.